manifests/env/nostromo-stage/telegraf/telegraf.conf

```diff
@@ -102,13 +86,12 @@ tags = ["CurrentLap", "CarClass"]
 
 [[outputs.influxdb_v2]]
 bucket = "racing"
+namepass = ["laps*"]
 organization = "b4mad"
 token = "$INFLUX_TOKEN"
 urls = ["http://influxdb2:8086"]
-namepass = ["laps*"]
 
 # Configuration for the Prometheus client to spawn
 [[outputs.prometheus_client]]
-## Address to listen on.
 listen = ":9090"
-namepass = ["internal_*"]
+namepass = ["*"]
```

Comment:
let's keep this. IIRC without this, we also expose the racing telemetry to prometheus